source/magpie/dataframe.d

```diff
@@ -1048,9 +1056,47 @@ public:
 
             return retrow;
         }
-
+        
```

Comment:
Its not great to add bits of whitespace, although its doesn't really matter.

Reply:
Apologies. Sometimes the cursor jumps on due to poor palm rejection on my laptom :sweat_smile: 
Fixed in cbe18d9 - addressed the complete linting issue